### ISSUE TITLE
Issues with drag-to-reorder

### ISSUE THREAD

**Issue by viiuan** [Kesomannen/gale#59]
I have always had this issue while using Gale. Dragging to reorder frequently results in a different mod than I was originally dragging being dragged and/or reordered, and it seems to happen more often if I try to reorder a mod so far that I scroll down. I originally thought it was an issue I was having with my mouse (long story- double clicks when single clicking, not holding down clicks) but it seems to be an issue with Gale.

**Comment by NakiriFox** [comment]
In fact, as long as you don't scroll the page when dragging the mod, this issue can be avoided.

However, it would be even better if the Mod could scroll through the page when dragging it.

**Comment by viiuan** [comment]
yeah exactly. i've just been not scrolling but it's annoying when you have a lot of mods installed

**Comment by VitrolicWither** [comment]
Came to github to submit an issue about this but since one is already open I will just comment that I am also have this problem when modding Risk of Rain 2.

**Comment by Kesomannen** [comment]
Should be much better with 0.9.0
